ldms/man/ldmsd_controller.man

```diff
@@ -334,10 +339,15 @@ attr=<value>
 .br
 A regular expression
 .TP
-.BI [interval " interval"]
+.BI [reconnect " interval"]
```

Comment:
I'll submit the change to support strings in another patch.

Reply:
Please put it in this patch. We are logically introducing a 'new' attribute or at least an alias with expanded syntax; this would be the place to get the documentation and the implementation all sync'd up. This patch is already pretty concise.